### PR TITLE
fix: long SubMenu title display

### LINKED ISSUE #22609: Long Submenu Title Being Overlayed By Arrow Icon
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://codepen.io/johnzhou/pen/NWqENOY](https://codepen.io/johnzhou/pen/NWqENOY)

### Steps to reproduce
Create a SubMenu with a long title.

### What is expected?
The SubMenu title and arrow icon should be clearly separated.

### What is actually happening?
The submenu arrow icon is being overlayed over the title.

| Environment | Info |
|---|---|
| antd | 4.0.4 |
| React | 16.13.1 |
| System | Microsoft Windows 10 Pro Version	10.0.17763 Build 17763 |
| Browser | Chrome 80.0.3987.132 |

---
I know that this can fixed in multiple ways, either adding a fixed width to the SubMenu component or removing the width property from the icon but I would prefer a fix in ant.design because I don't think having a long title is out of the ordinary.

### PR DESCRIPTION
**Pull request by wwyx778**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close #22609

### 💡 Background and solution
Fix Long Submenu Title Being Overlayed By Arrow Icon

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/menu/style/index.less [hunk 1]
```diff
@@ -396,7 +396,7 @@
       height: @menu-item-height;
       margin-top: @menu-item-vertical-margin;
       margin-bottom: @menu-item-vertical-margin;
-      padding: 0 16px;
+      padding: 0 34px 0 16px;
```
Comment (xrkffgg) [inline components/menu/style/index.less:399]: 不建议在公共样式中修改，可单独处理这个。

顺便 rebase 下 master

**Review by xrkffgg — CHANGES_REQUESTED** [review]

**Comment by yoyo837** [comment]
Commits should be clean.

**Comment by wwyx778** [comment]
> Commits should be clean.

我可以关掉这个PR重新来一次吗？

**Comment by yoyo837** [comment]
先fetch最新rebase 到 你的分支，如果是后面同步最新的可能需要`--force` push

**Comment by wwyx778** [comment]
> 先fetch最新rebase 到 你的分支，如果是后面同步最新的可能需要`--force` push

好的。谢谢。

**Pull request closed without merging**